UefiPayloadPkg/PayloadLoaderPeim/FitPayloadLoaderPeim.inf

```diff
@@ -35,25 +35,33 @@
 
 [LibraryClasses]
   PcdLib
-  MemoryAllocationLib
   BaseMemoryLib
   PeiServicesLib
   HobLib
   BaseLib
   PeimEntryPoint
   DebugLib
   FdtLib
+##for testing, remove this for HandOffFdtEnable == FALSE scenario: BuildFdtLib

```

Comment:
let use some comments like below example to guide users what to do for this bootloader module:
## Hook this lib to FitPayloadLoaderPeim.inf in platform DSC when HandOffFdtEnable == TRUE.

Reply:
resolved